pages/settings/SettingsWallet.qml

```diff
@@ -47,20 +48,75 @@ Rectangle {
         anchors.right: parent.right
         anchors.margins: 20
         anchors.topMargin: 0
-        spacing: 8
+        spacing: 0
 
         MoneroComponents.SettingsListItem {
-            buttonText: qsTr("Close wallet") + translationManager.emptyString
-            description: qsTr("Logs out of this wallet.") + translationManager.emptyString
+            iconText: FontAwesome.signOutAlt
             title: qsTr("Close this wallet") + translationManager.emptyString
+            description: qsTr("Logs out of this wallet.") + translationManager.emptyString
 
             onClicked: appWindow.showWizard()
         }
 
         MoneroComponents.SettingsListItem {
-            buttonText: qsTr("Create wallet") + translationManager.emptyString
-            description: qsTr("Creates a new wallet that can only view and initiate transactions, but requires a spendable wallet to sign transactions before sending.") + translationManager.emptyString
+            iconText: FontAwesome.lock
+            title: qsTr("Lock this wallet") + translationManager.emptyString
+            description: qsTr("Enter your password to unlock it.") + translationManager.emptyString
```

Comment:
```suggestion
            description: qsTr("Locks you wallet and asks for the password to unlock it.") + translationManager.emptyString
```

Reply:
Most items' descriptions describe what is the action of the button. We should keep this consistent.